packages/ndla-image-search/src/ImageSearch.js

```diff
@@ -414,6 +414,7 @@ class ImageSearch extends React.Component {
           query={queryObject}
           onClick={this.searchImages}
           pageItemComponentClass="button"
+          type="button"
```

Comment:
Benyttes Pager kun her? Er det dermed gitt at denne propen finnes?

Reply:
Godt poeng, sjekka raskt og nei den brukes selvfølgelig flere steder. Jeg kan legge på en sjekk for om den er tilstede.

Reply:
Så vidt jeg kan se så vil `type` bli ignorert dersom proppen er null eller ikke eksisterer, så lenge den ikke er satt til `isRequired`. Så jeg lener egentlig litt mot å ha det slik det er nå.

Om noen har noen gode argumenter mot det, så kan jeg endre mening.